tools/theme-preview.sh

```diff
@@ -2,31 +2,84 @@
 #shellcheck disable=SC2317
 set -u
 
-COLUMNS=160
-
-. ./liquidprompt --no-activate
-
-# If a theme was given, use the theme and source all other arguments.
-if [[ -z ${1-} || $1 == --help ]]; then
+usage() {
   printf '
-Usage: %s theme [sourced files...]
+Usage: %s theme [options...] [sourced files...]
 
 Print out example prompts based on a standard set of input conditions. Designed
 to showcase Liquid Prompt themes.
 
-Example usage: %s powerline themes/powerline/powerline.theme
+Options:
+  --config-file=<filename> Load <filename> as an additional Liquid Prompt
+                           config file.
+  --user-config            Load the user, or if not found, the system config
+                           file.
+  --reproducible           Set the terminal size to a static value. Load no
+                           config files. Overrides any --config-file or
+                           --user-config arguments.
+  -h, --help               Print this help text.
+
+Example usage: %s unfold themes/unfold/unfold.theme
```

Comment:
I think we need at least an example with _reproducible_, and one with config files, to show how to load presets.

Reply:
```suggestion
Example usage:
%s alternate_vcs themes/alternate_vcs/alternate_vcs.theme
%s unfold themes/unfold/unfold.theme --reproducible
%s unfold themes/unfold/unfold.theme --user-config --config-file contrib/presets/colors/256-colors-dark.conf
%s 
```